webapp/channels/src/components/youtube_video/youtube_video.tsx

```diff
@@ -71,59 +110,64 @@ export default class YoutubeVideo extends React.PureComponent<Props, State> {
-                    allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share'
-                    referrerPolicy={this.props.youtubeReferrerPolicy ? 'strict-origin-when-cross-origin' : 'no-referrer'}
-                    title={videoTitle}
-                    sandbox='allow-scripts allow-same-origin allow-popups allow-presentation'
-                />
+                <div className='video-playing'>
+                    <iframe
+                        src={`https://www.youtube.com/embed/${videoId}?autoplay=1&rel=0${time}`}
+                        title={videoTitle}
+                        width='100%'
+                        height='100%'
+                        frameBorder='0'
+                        allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture'
+                        allowFullScreen={true}
+                        referrerPolicy={this.props.youtubeReferrerPolicy ? 'origin' : undefined}
+                        sandbox='allow-scripts allow-same-origin allow-popups allow-presentation'
+                    />
+                </div>
             );
         } else {
-            const image = metadata?.images[0];
-
             content = (
-                <div className='embed-responsive video-div__placeholder'>
-                    <div className='video-thumbnail__container'>
-                        <ExternalImage src={image?.secure_url || image?.url || ''}>
-                            {(safeUrl) => (
-                                <img
-                                    src={safeUrl}
-                                    alt='youtube video thumbnail'
-                                    className='video-thumbnail'
-                                />
-                            )}
-                        </ExternalImage>
-                        <div className='block'>
-                            <span className='play-button'><span/></span>
-                        </div>
+                <div
+                    className='video-thumbnail__container'
+                    onClick={this.play}
+                    role='button'
+                    aria-label={`Play ${videoTitle} on YouTube`}
+                    tabIndex={0}
+                    onKeyDown={(e) => {
+                        if (e.key === 'Enter' || e.key === ' ') {
+                            e.preventDefault();
+                            this.play();
+                        }
+                    }}
+                >
+                    <img
+                        className='video-thumbnail'
+                        src={this.state.thumbnailUrl}
+                        alt={`Thumbnail for ${videoTitle} on YouTube`}
+                        onError={this.handleImageError}
+                    />
+                    <div
+                        className='play-button'
+                        role='img'
+                        aria-label='Play video'
```

Comment:
I think these are can be replaced with `role='presentation'` since this information is redundant for a user using a screen reader since they already know that this is a button with the name "Play Some Video on YouTube".
```suggestion
                        role='presentation'
```

The alt text for the thumbnail image is also sort of redundant, but I guess it could be useful if the user were using a tool that listed all of the visible images for whatever reason